pkg/olm/olm.go

```diff
@@ -205,6 +205,18 @@ func GenerateCSV(opConf *operator.Conf) *operv1.ClusterServiceVersion {
 	}
 	uiText := []string{"urn:alm:descriptor:com.tectonic.ui:text"}
 	uiResources := []string{"urn:alm:descriptor:com.tectonic.ui:resourceRequirements"}
+	uiFieldGroup := "urn:alm:descriptor:com.tectonic.ui:fieldGroup:"
+	uiK8sSecret := "urn:alm:descriptor:io.kubernetes:Secret"
+	uiBooleanSwitch := "urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
+	uiNumber := "urn:alm:descriptor:com.tectonic.ui:number"
+	uiFieldGroupText := "urn:alm:descriptor:com.tectonic.ui:text"
```

Comment:
This is not a fieldGroup right? 
Lets rename it to uiText and since few lines before we defined uiText but in that case it was as an array, then I suggest to replace the other uses of uiText to `[]string{uiText}`.
```suggestion
	const uiText = uiTectonic + "text"
```